zeton/auth.py

```diff
@@ -41,6 +42,18 @@ def logout():
     return redirect(url_for('auth.login'))
 
 
+@bp.route('/register', methods=['GET', 'POST'])
```

Comment:
w metodach wystarczy sam `GET`

Reply:
Moje niedopatrzenie - to pozostałość po poprzednim sposobie rejestracji z poziomu widoku, który akceptował obie metody. Zanotowane do usunięcia :)